exif.js

```diff
@@ -928,10 +928,10 @@
     }
 
     EXIF.getData = function(img, callback) {
-        if ((self.Image && img instanceof self.Image)
-            || (self.HTMLImageElement && img instanceof self.HTMLImageElement)
-            && !img.complete)
-            return false;
+        const isImage = (self.Image && img instanceof self.Image)
```

Comment:
`const` is now standard and best practice for variables declaration. And it's widely supported even without babel. http://caniuse.com/#search=const